napari/layers/base/base.py

```diff
@@ -1624,13 +1625,44 @@ def get_status(
         msg : string
             String containing a message that can be used as a status update.
         """
-        value = self.get_value(
-            position,
-            view_direction=view_direction,
-            dims_displayed=dims_displayed,
-            world=world,
-        )
-        return generate_layer_status(self.name, position, value)
+        if position is not None:
+            value = self.get_value(
+                position,
+                view_direction=view_direction,
+                dims_displayed=dims_displayed,
+                world=world,
+            )
+        else:
+            value = None
+
+        if self.source.reader_plugin:
+            try:
+                layer_base = os.path.basename(self.source.path)
+            except KeyError:
+                pass
+            name = trans._(
+                '{layer_base},  source: {source} (plugin)',
+                layer_base=layer_base,
+                source=self.source.reader_plugin,
+            )
```

Comment:
Potentially undefined variable usage? I do not see the previous `layer_base` definition that could be used if `KeyError` occur.